code/datums/diseases/advance/presets.dm

```diff
@@ -1,10 +1,16 @@
 // Cold
+/datum/disease/advance/cold
+	copy_type = /datum/disease/advance
+

```

Comment:
This and the one below it are not strictly necessary, because both the symptoms list and the name var are overwritten in the Copy() proc after New() is called, but it is silly to have some disease datums have different types because they came from one of these convenient-initializer subtypes.